packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,60 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    userIsAdmin: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId()},
+        $in: {adminOfOrgs: [@organizationId]}
+      )
```

Comment:
userIsMember determines whether the Join button shows up, and userIsAdmin determines whether the Make/Remove Admin buttons show up